From cd1d036169fedbc15834d7d4279789a863ec450d Mon Sep 17 00:00:00 2001
From: Benjamin Tissoires <benjamin.tissoires@redhat.com>
Date: Tue, 16 Sep 2014 17:41:01 -0400
Subject: [redhat] configs: enable elo module

Message-id: <1410889261-29971-1-git-send-email-benjamin.tissoires@redhat.com>
Patchwork-id: 95658
O-Subject: [RHEL7.1 PATCH] redhat/configs: enable elo module
Bugzilla: 1140214
RH-Acked-by: Peter Hutterer <peter.hutterer@redhat.com>
RH-Acked-by: Aristeu Rozanski <aris@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1140214
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=7968873
Upstream: Red Hat only
Tested by the customer directly (see #1140214)

Commit ee7274258ce3742df41686728a07ec194671cf41 disabled a bunch
of touchscreen drivers. We have one customer requesting elo.ko,
so re-enable it.

CONFIG_TOUCHSCREEN_ELO=m

Signed-off-by: Benjamin Tissoires <benjamin.tissoires@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_TOUCHSCREEN_ELO b/redhat/configs/generic/CONFIG_TOUCHSCREEN_ELO
index 043c220..889c6b2 100644
--- a/redhat/configs/generic/CONFIG_TOUCHSCREEN_ELO
+++ b/redhat/configs/generic/CONFIG_TOUCHSCREEN_ELO
@@ -1 +1 @@
-# CONFIG_TOUCHSCREEN_ELO is not set
+CONFIG_TOUCHSCREEN_ELO=m
-- 
1.7.1